scripts/workflow.treering.R

```diff
@@ -24,31 +26,38 @@ library(PEcAn.visualization)
 
 #---------------- Load PEcAn settings file. -------------------------------------------------------#
 # Open and read in settings file for PEcAn run.
-settings <- read.settings("~/demo.sda/demo.xml")
+settings <- read.settings("/fs/data2/output//PEcAn_1000001249/pecan.xml")
```

Comment:
A lot of the changes to workflow.treering.R seem branch specific. Not sure we want these pushed back to the mainline where they'll break the demo.